meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk.bb

```diff
@@ -46,6 +47,7 @@ RDEPENDS_${PN} = " \
     systemd \
     curl \
     resin-device-uuid \
+    jq \
```

Comment:
Yeah, I had added it because I used it in the start-resin-supervisor script, but it's not necessary anymore.  Removed.